os/drivers/serial/Kconfig

```diff
@@ -3,6 +3,26 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
+config DYNAMIC_LOGS_STORING
+	bool "dynamic storing of logs"
+	default n
+	depends on DEBUG_MM_HEAPINFO
+
+config LOG_CHUNK_SIZE
+	int "dynamic log chunk size"
+	default 1024
+	depends on DYNAMIC_LOGS_STORING
+
+config MAX_LOG_SIZE_KB
```

Comment:
Instead of MAX_SIZE, you should use 'number of buffer'.
For example if user set chunk size as a 2048 but max size is 1k, it will make an error.